operator/hack/lokistack_gateway_ocp.yaml

```diff
@@ -3,11 +3,19 @@ kind: LokiStack
 metadata:
   name: lokistack-dev
 spec:
-  size: 1x.extra-small
+  size: 1x.small
```

Comment:
Is this an intended change? It will make our testing deployments much larger...

Reply:
Nope this change is a mistake. Will return this back to 1x.extra-small